pkg/controller/elasticsearch/version/zen2/voting_exclusions.go

```diff
@@ -19,6 +20,35 @@ var (
 	log = logf.Log.WithName("zen2")
 )
 
+const (
+	// VotingConfigExclusionsAnnotationName is an annotation that stores the last applied voting config exclusions.
+	// An empty value means no voting config exclusions is set.
+	VotingConfigExclusionsAnnotationName = "elasticsearch.k8s.elastic.co/voting-config-exclusions"
+)
+
+func serializeExcludedNodesForAnnotation(excludedNodes []string) string {
+	return strings.Join(excludedNodes, ",")
```

Comment:
Should this be sorted to ensure a stable comparison? I know we have an implicitly stable ordering due to the way we calculate downscales. So there would be no difference in practice as long as the downscale calculation stays the same.

Reply:
good point, it feels safer to sort here even though currently useless